docker-compose.yml

```diff
@@ -39,9 +42,64 @@ services:
         extra_hosts:
             # just if you have a local CKAN instance running in _nginx_ host in you local machine
             # export HOST_IP=`ip -4 addr show scope global dev docker0 | grep inet | awk '{print \$2}' | cut -d / -f 1`
-            - "nginx:${HOST_IP}"
+            - "ckan:${HOST_IP}"
         healthcheck:
             test: ["CMD-SHELL", "[ -f /usr/local/airflow/airflow-webserver.pid ]"]
             interval: 30s
             timeout: 30s
             retries: 3
+
+    ckan-dev:
+        build:
+            context: ckan/
+            dockerfile: Dockerfile.dev
+            args:
+                - TZ=${TZ}
+        env_file:
+            - .env
+        links:
+            - db
+            - solr
+            - redis
+            - datapusher
+        ports:
+            - "0.0.0.0:${CKAN_PORT}:5000"
+        volumes:
+            - ./src:/srv/app/src_extensions
+            - ckan_storage:/var/lib/ckan
+            # debug extensions
+            # - /home/user/exts-folder/ckanext-xxx:/srv/app/src_extensions/ckanext-xxx
+        
+    datapusher:
+        container_name: datapusher
+        build:
+            context: datapusher/
+        ports:
+            - "8800:8800"
+    
+    db:
```

Comment:
what is the difference between `db` and `postgres`? please use more meaningful names

Reply:
Moved to airflow_db and ckan_db [here](https://github.com/datopian/ckan-ng-harvest/commit/cb767e1195662a6cb9800cbe7e8701df2cacf261)